internal/models/factor.go

```diff
@@ -121,6 +127,7 @@ type Factor struct {
 	Secret       string      `json:"-" db:"secret"`
 	FactorType   string      `json:"factor_type" db:"factor_type"`
 	Challenge    []Challenge `json:"-" has_many:"challenges"`
+	PhoneNumber  *string     `json:"phone_number" db:"phone_number"`
```

Comment:
can we use `storage.NullString` instead? it's easier to deal with because you don't have to check for nil and "", you just need to check for nil. 
```suggestion
	PhoneNumber  storage.NullString     `json:"phone" db:"phone"`
```

 maybe also rename it to just `Phone` for consistency?

Reply:
Yeah that's fine too. Actually thinking of renaming it to MFA (Phone) and adjusting the method to be `phone` since we allow for WhatsApp for as a channel